packages/react-sdk/src/components/image.ws.tsx

```diff
@@ -26,6 +52,44 @@ const meta: WsComponentMeta<typeof Image> = {
   isInlineOnly: false,
   isListed: true,
   label: "Image",
+  transformAsset: (assetProp, props) => {
```

Comment:
now that makes meta non-serializable, not sure this is not a problem

Reply:
discussed: we want a universal transformers interface that can be sticked together like middleware. This one is not universal right now